core/tests/test_utils.py

```diff
@@ -144,12 +163,27 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Tests if unicode string are working properly.
```

Comment:
That's not what this does. Please read the method carefully and take a look at how it's used -- it doesn't, on its own, perform a test.

Reply:
Done!